src/element.ts

```diff
@@ -36,27 +56,28 @@ export class Element {
   }
 
   // set music font
-  setFontStack(fontStack) {
+  setFontStack(fontStack: Font[]): this {
     this.fontStack = fontStack;
     this.musicFont = fontStack[0];
     return this;
   }
-  getFontStack() {
+  getFontStack(): Font[] {
```

Comment:
Line break before would be consistent with the rest of the file

Reply:
Do not worry, this is included in the 2nd PR with the comments.